assets/src/scripts/collapse-sections.js

```diff
@@ -52,7 +52,18 @@ document.addEventListener( 'DOMContentLoaded', function () {
 				elem.nextElementSibling &&
 				elem.nextElementSibling.tagName !== 'H1' &&
 				! elem.nextElementSibling.classList.contains( 'nav-reading--page' ) &&
-				! ( elem.nextElementSibling.tagName === 'DIV' && elem.nextElementSibling.className === 'glossary' )
+				! (
+					(
+						elem.nextElementSibling.tagName === 'DIV' &&
+						(
+							elem.nextElementSibling.className === 'glossary' ||
+							elem.nextElementSibling.className === 'contributors' ||
+							elem.nextElementSibling.className === 'footnotes' ||
```

Comment:
Not sure the `footnotes` class check is necessary here since we always have a `before-footnotes` hr element before.